core/cloudflow-operator/src/main/scala/cloudflow/operator/action/Action.scala

```diff
@@ -106,9 +117,9 @@ object Action {
  */
 class CreateAction[T <: ObjectResource](
     val resource: T,
-    val format: Format[T],
-    val resourceDefinition: ResourceDefinition[T],
-    val editor: ObjectEditor[T]
+    implicit val format: Format[T],
```

Comment:
moving these implicits here cleaned up the code nicely! :+1:

Reply:
Thanks!